frontend/components/Status.tsx

```diff
@@ -13,7 +13,7 @@ const Status = ({
   className,
 }: {
   variant?: Variant | undefined;
-  children: React.ReactNode;
+  children?: React.ReactNode;
```

Comment:
```suggestion
  children: React.ReactNode;
```
@nazargladish maybe we should it as it.